.pylintrc

```diff
@@ -54,7 +54,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=missing-docstring
+disable=missing-docstring, C0103
```

Comment:
```
tfd = tfp.distributions
tfb = tfp.distributions.bijectors
ed = tfp.edward2
```
Since I am using this and not using all-caps, its causing errors. I am unable to find a different way to import tfd, tfb and ed. Should I selectively ignore it for this page?